shell/src/main/java/alluxio/cli/fs/command/RmCommand.java

```diff
@@ -86,18 +97,30 @@ protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioExcepti
     if (cl.hasOption(REMOVE_UNCHECKED_OPTION_CHAR)) {
       options.setUnchecked(true);
     }
+    boolean isAlluxioOnly = cl.hasOption(REMOVE_ALLUXIO_ONLY.getOpt());
+    if (isAlluxioOnly) {
```

Comment:
nit: the `if` statement is unnecessary, we can always do `options.setAlluxioOnly(isAlluxioOnly)`